profile/README.md

```diff
@@ -0,0 +1,70 @@
+
+## 🔨 Transparency.dev Projects
+
+This repository hosts critical projects for building end-to-end transparency ecosystems, including:
+- [trillian-tessera](https://github.com/transparency-dev/trillian-tessera) - a Go library for building tile-based transparency logs
+- [static-ct](https://github.com/transparency-dev/static-ct) - An implementation of the Static CT API based on Tessera
+- [witness](https://github.com/transparency-dev/witness) - Libraries and binaries for running witnesses for verifiable logs
+- [armored-witness](https://github.com/transparency-dev/armored-witness) - A plug and play witness solution intended to kick-start a cross-ecosystem witness network
+
+## ⚒️ Related Projects
+
+Explore a variety of related transparency projects through the following links:
+
+- [trillian](https://github.com/google/trillian) - A transparent, highly scalable and cryptographically verifiable data store.
+- [C2SP](https://github.com/C2SP/C2SP) - Community Cryptography Specification Project, home to transparency log specifications
+- [sunlight](https://github.com/FiloSottile/sunlight) - A Certificate Transparency log implementation and monitoring API designed for scalability, ease of operation, and reduced cost.
+
+## 🌳 Roadmap
+
+Here's a glimpse of some of the things we're working on during the upcoming months, along with highlights of recently completed initiatives.
+For more details, check out the links or join us on [Slack](https://join.slack.com/t/transparency-dev/shared_invite/zt-27pkqo21d-okUFhur7YZ0rFoJVIOPznQ) to discuss. 
+
+### ✅ Recently Completed
+
+- [Trillian 1.7.0](https://github.com/google/trillian/releases/tag/v1.7.0) with PostgreSQL support
+
+### ⚙️ Now (well defined with committed development resources)
+- [Trillian Tessera Alpha](https://github.com/transparency-dev/trillian-tessera/issues/1) release
+
+### 🏗️ Next (design stage with some assumptions to validate before proceeding)
+- [Static CT API Tessera personality](https://github.com/transparency-dev/static-ct/issues/10)
+- Bastion support in Tessera
+- Migration resources for Tiled logs
+
+### 💡 Later (aspirational items, not yet scoped)
+- Productionize the Witness Network
+- Transparency.dev Summit 2025
+
+  
+## 💗 Transparency.dev Community
+
+Whether you are a seasoned developer or just getting started, there’s a place for you in the community. We welcome contributors and are thrilled to have you join our community as we build modern transparency ecosystems for building trust in every domain. Get involved here:
+
+### 💝 How to Contribute
+
+There are several ways you can contribute to our open source projects:
+- Submit issues related to bugs or new feature requests
+- Submit pull requests by following guidance in each project's [Contributing Guidelines](https://github.com/transparency-dev/trillian-tessera/blob/main/CONTRIBUTING.md)
+- Or simply ⭐️ the repository to show your appreciation
```

Comment:
nit: is this going to be shown in a repo people can star, or at https://github.com/transparency-dev/? Maybe replace "repository" with "repositories"?

Reply:
Good idea, fixed in latest commit.